augly/video/transforms.py

```diff
@@ -801,10 +859,7 @@ def apply_transform(
 
 class HStack(BaseTransform):
     def __init__(
-        self,
-        second_video_path: str,
-        use_second_audio: bool = False,
-        p: float = 1.0,
+        self, second_video_path: str, use_second_audio: bool = False, p: float = 1.0,
```

Comment:
In the future, please avoid making formatting changes to other functions not relevant to your PR. I'm guessing what happened is that you ran `black` on your changed files. But we actually have a TODO to run `black` on **all** of our files which we will do in another PR, because due to a weird config situation internally `black` wasn't being run.

Reply:
```suggestion
        self, second_video_path: str, use_second_audio: bool = False, p: float = 1.0
```

Reply:
this is not resolved, please make this change!